README.md

```diff
@@ -8,20 +8,40 @@
 [![CircleCI Build Status](https://circleci.com/gh/universal-ctags/ctags.svg?style=shield&circle-token=2e582261da84ebc6d21725b05381f410bc5de29d)](https://circleci.com/gh/universal-ctags)
 ![GitHub Actions/VALGRIND](https://github.com/universal-ctags/ctags/workflows/run%20units%20target%20under%20VALGRIND/badge.svg)
 
-Universal Ctags generates an index (or tag) file of language objects found in source files for many popular programming languages. This index makes it easy for text editors and other tools to locate the indexed items. Universal Ctags improves on traditional ctags because of its multilanguage support, its ability for the user to define new languages searched by regular expressions, and its ability to generate emacs-style TAGS files.
-
-universal-ctags has the objective of continuing the development from
-what existed in the Sourceforge area. Github exuberant-ctags
-repository was started by Reza Jelveh and was later moved to the
-universal-ctags organization.
-
-The goal of the project is preparing and maintaining common/unified working
-space where people interested in making ctags better can work
-together.
+[Universal Ctags](https://ctags.io/) (abbreviated as u-ctags) is an
+implementation of `ctags`. 
+`ctags` generates an index (or tag) file of language objects found in source
+files for programming languages.
+This index makes it easy for text editors and other tools to locate the indexed
+items.
+
+Darren Hiebert's [Exuberant Ctags](http://ctags.sourceforge.net/) (e-ctags), the
```

Comment:
In the e-ctags project, there were some important developers. So I would like to say
"[Exuberant Ctags](http://ctags.sourceforge.net/) (e-ctags) maintained by Darren Hiebert, ..."

Reply:
Fixed